src/loader.js

```diff
@@ -3,6 +3,53 @@ import path from 'path';
 import gParser from 'gettext-parser';
```

Comment:
file `loader.js` is used on a backend. We should be able to transform our translations object on a frontend. I think we should put all transform logic inside `addLocale` method.